src/layer.js

```diff
@@ -107,9 +125,34 @@ function sigmoid(value) {
 
 /**
  * Function that calculates the derivate of the sigmoid function.
+ * given the value of the sigmoid function at that point
  * @param value
  * @returns {number}
  */
 function sigmoidGradient(value) {
     return value * (1 - value);
 }
+
+/**
+ * Function that caclulates the hyperbolic tangent (tanh) function at some value
+ * @param value
+ * @returns {number}
+**/
+
+function tanh(value){
+    return Math.tanh(value);
+}
+
+/**
+ * Function that caclulates the derivative of 
+ * hyperbolic tangent (tanh) function given the value
+ * of the hyperbolic tangent function at that point
+ * @param value
+ * @returns {number}
+**/
+
+function tanhGradient(value){
+    return 1-Math.pow(value, 2)
```

Comment:
some minimal things like adding space arround operators signs so for example here you'll change this for `1 - Math`

It's easier just to check https://github.com/cheminfo/eslint-config for all the rules